arex-instrumentation-foundation/src/main/java/io/arex/foundation/model/ConfigQueryResponse.java

```diff
@@ -60,7 +60,7 @@ public static class ResponseBody {
         private ServiceCollectConfig serviceCollectConfiguration;
         private int status;
         private List<DynamicClassConfiguration> dynamicClassConfigurationList;
-        private String targetAddress;
+        private boolean agentEnabled;
```

Comment:
Suggest to keep the `targetAddress` and just add `agentEnabled`, `message` fields